README.md

```diff
@@ -32,6 +32,8 @@ https://github.com/Mr0grog/editorconfig-textmate/releases/latest
 
 Un-tar it and double-click the `editorconfig-textmate.tmplugin` file to install. You can also manually drag the `.tmplugin` package into `~/Library/Application Support/TextMate/PlugIns/` in Finder.
 
+**Note:** The PlugIn is loaded after restarting TextMate and must not be activated manually.
+It will not be listed in the Bundles list since Bundle != PlugIn in TextMate.
```

Comment:
I’m not even sure how you *would* activate a plug-in manually. Maybe clearer to say “the plug-in will be loaded automatically after restarting TextMate. It will not be listed in the bundles list, since bundles and plug-ins are different in TextMate.” ?